homeassistant/components/shelly/light.py

```diff
@@ -152,14 +156,21 @@ def hs_color(self) -> tuple[float, float]:
             red = self.block.red
             green = self.block.green
             blue = self.block.blue
-        return color_RGB_to_hs(red, green, blue)
+        return [red, green, blue]
```

Comment:
This should return a tuple
```suggestion
        return (red, green, blue)
```